app/assets/javascripts/controllers/playbook-reusable-code-mixin.js

```diff
@@ -141,6 +141,14 @@ function playbookReusableCodeMixin(API, $q, miqService) {
       );
     }
 
+    allApiPromises.push(API.get('/api/zones/?expand=resources&attributes=id,description' + sortOptions)
```

Comment:
Tiny detail here.. we're showing the `description` field, but `sortOptions` includes `sort_by=name`.

If those 2 don't match, zones won't be sorted right in the select.